Video/test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import a11yTestHelper from '../lib/a11yTestHelper';
+import Video from './index';
+
+describe('Video', () => {
+  it('should pass accessibility audit', (done) => {
+    a11yTestHelper.testComponent(
```

Comment:
after importing with es6 import then you can use `testComponent` directly :smile_cat: